tests/ut/conftest.py

```diff
@@ -38,3 +38,9 @@ def get_lock_file() -> Config:
 def mock_subprocess_run() -> Generator[MagicMock, None, None]:
     with patch("subprocess.run") as mock_subprocess_run:
         yield mock_subprocess_run
+
+
+@pytest.fixture
+def mock_subprocess_check_call() -> Generator[MagicMock, None, None]:
```

Comment:
Are you aware of `pytest-subprocess` and its `fake_process` (alias `fp`) fixture? It seems to be a more standard way to mock subprocess calls and we're already using it in other tests.

Reply:
Fixture mock_subprocess_check_call removed. Test cases rework using fake_process fixture.